README.rst

```diff
@@ -133,15 +123,22 @@ More examples
 
 .. code-block:: python
 
-    >>> import boto3
+    >>> import os, sys, boto3
+    >>>
+    >>> def get_session():
+    ...     # This is here so we can doctest this example under TravisCI
```

Comment:
Mild -1: the purpose of README is clear communication, not doctests.

If the two goals clash, clarity of message takes precedence. Doctests less important.

Reply:
btw if we have broken example in the docstring, that's bad for users, for this reason, I propose to make something similar as in gensim we done: some "internal util functions" inside package that "make dirty work" and just re-use them in docstring (similar stuff in gensim are `datapath`, small corpora & dictionaries, etc).